### DISCUSSION TITLE
Welcome to OpenFL Discussions!

### DISCUSSION THREAD

**Discussion by psfoley** [securefederatedai/openfederatedlearning#109] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by changhongyan123** [comment]
Hi all, 

I am a third-year phd student under the supervision of Prof. Reza Shokri. My research focuses on privacy, fairness, and federated learning. We would like to integrate the privacy meter tool to OpenFL for measuring the information leakage of parties' private dataset during the FL
